modules/models/behavior/static_trajectory/behavior_static_trajectory.cpp

```diff
@@ -61,7 +61,9 @@ std::pair<int, int> BehaviorStaticTrajectory::Interpolate(
   StateRowVector delta;
   double alpha;
   int idx = -1;
-  BARK_EXPECT_TRUE(static_trajectory_.rows() > 1);
+  if (static_trajectory_.rows() < 2) {
+    return {-1, -1};
+  }
```

Comment:
was this necessary since we introduced the expired status flag? This might make errors in the dataset harder to debug?

Reply:
Either we have to somehow handle this in the behavior model or we have to skip agents in the scenario generation that only have one trajectory point.